src/client/containers/useUpdateBallots.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+export function useUpdateBallots(userId, boardId, candidatesBody) {
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    (async () => {
+      const response = await fetch(
+        `/api/users/${userId}/boards/${boardId}/ballots`,
+        {
+          method: 'PUT',
+          mode: 'cors',
+          headers: {
+            'Content-type': 'application/json',
+          },
+          body: JSON.stringify(candidatesBody),
+        },
+      );
+      if (!response.ok) {
+        setError(
+          `Error updating ballot: ${response.status}. ${response.statusText}`,
+        );
+      }
+    })();
+  }, [userId, boardId, candidatesBody]);
+
+  return { error };
```

Comment:
Same problem here. What should the dependency array hold here and is this return correct as it only returns error?

Reply:
You don't need a useEffect here. Just return a method to update the ballots. The method will take the current array of candidate sortings